onfido_strings.translatable

```diff
@@ -386,5 +395,6 @@
   "loading": "$loading$",
   "back": "$back$",
   "cancel": "$cancel$",
-  "close": "$close$"
+  "close": "$close$",
+  "continue": "$continue$"
```

Comment:
Will need to update `MIGRATION.md` doc with new key(s) added. Not sure if for iOS SDK you guys also include the new keys added for a new screen like the case here with the new keys in lines 102-108?

Reply:
added to migration doc